example-3.0/server/config.json

```diff
@@ -1,10 +1,25 @@
 {
-  "legacyExplorer": false,
   "restApiRoot": "/api",
   "host": "0.0.0.0",
   "port": 3000,
-  "url": "http://localhost:3000/",
   "remoting": {
-    "cors": false
-  }
+    "context": {
+      "enableHttpContext": false
+    },
```

Comment:
Here is the config template:
https://github.com/strongloop/loopback-workspace/blob/master/server/config.json